### DISCUSSION TITLE
See tags in the list of commits

### DISCUSSION THREAD

**Discussion by tomasMizera** [community/community#70298] [Repositories]
### Select Topic Area

Product Feedback

### Body

I bet this was requested quite a few times, but would it be possible to see tags associated with commits when listing commits? GitLab has it nicely.

**Github:**
![image](https://github.com/community/community/assets/22449698/ef6fdb8f-7354-4f42-9058-91f2c3ec1aa6)

**GitLab:**
![image](https://github.com/community/community/assets/22449698/96089610-ae94-4666-a173-2d954130fa04)

**Comment by Marcono1234** [comment]
Sounds like a duplicate of #58544
